module/input/NatNet/src/NatNet.cpp

```diff
@@ -307,37 +329,65 @@ namespace module::input {
         emit(std::move(mocap));
     }
 
-    void NatNet::processModel(const Packet& packet) {
+    void NatNet::process_model(const Packet& packet) {
+        // Remove old definitions
+        marker_set_models.clear();
+        rigid_body_models.clear();
+        skeleton_models.clear();
+        force_plate_models.clear();
+        device_models.clear();
+        camera_models.clear();
 
+        std::stringstream ss;
```

Comment:
tbh this is scope creep-y, you can revert and make an issue instead (and ideally address the issue in your next PR :stuck_out_tongue: )

Reply:
Is this variable even used?

Reply:
No worries, I'll do that now.

I think it used to be used for storing some part of one of the models but its redundant now so I'll take it out.